resources/input/fsh/profiles/VitalSignDE_Atemfrequenz.fsh

```diff
@@ -10,11 +10,11 @@ Id: observation-de-vitalsign-atemfrequenz
 Instance: Example-observation-atemfrequenz
 InstanceOf: VitalSignDE_Atemfrequenz
 Usage: #example
-* meta.profile[0] = "http://hl7.org/fhir/StructureDefinition/resprate"
+* meta.profile[0] = "http://hl7.org/fhir/StructureDefinition/vitalsigns"
```

Comment:
Wenn wir die Konformität zum internationalen Vitalsigns Profil hinzufügen, sollten wir es in `VitalSignDE hinzufügen`.

Reply:
Das ist hier im Beispiel, beide Canonicals sollten dort vorhanden sein.